disnake/ext/commands/cooldowns.py

```diff
@@ -27,28 +27,15 @@
 
 
 class BucketType(int, Enum):
-    """Specifies a type of bucket for e.g. a cooldown."""
-
     default = 0
-    """The default bucket operates on a global basis."""
     user = 1
-    """The user bucket operates on a per-user basis."""
     guild = 2
-    """The guild bucket operates on a per-guild basis."""
     channel = 3
-    """The channel bucket operates on a per-channel basis."""
     member = 4
-    """The member bucket operates on a per-member basis."""
     category = 5
-    """The category bucket operates on a per-category basis."""
     role = 6
-    """The role bucket operates on a per-role basis.
-
-    New in version 1.3.
-    """
 
     def get_key(self, msg: Message) -> Any:
-        """Create a key with this bucket type for the provided message."""
```

Comment:
this docstring should probably stay.

Reply:
On that note, the same goes for _all_ of these comments in which you ask me to "restore" stuff.

Reply:
IMO that should be here since it isn't an enum member.

Reply:
Sure, as long as that isn't considered scope creep, I'm down to add it in this PR.

Reply:
it's not considered scope creep, imo